public/index.html

```diff
@@ -187,13 +187,13 @@ <h3>LET'S CONNECT</h3>
             <li><a href="#"><img src="../assets/social icons/Linkedin icon.svg" alt=""></a></li>
             <li><a href="#"><img src="../assets/social icons/Vector (1).svg" alt=""></a></li>
             <li><a href="#"><img src="../assets/social icons/Vector (2).svg" alt=""></a></li>
-            <li><a href="#"><img src="../assets/social icons/Vector (3).svg" alt=""></a></li>
+            <li><a href="#"><imgpublic/index.html src="../assets/social icons/Vector (3).svg" alt=""></a></li>
```

Comment:
- [ ] Great job on removing the list styling from the social icons, however, as requested by the previous reviewer please make sure your icons look like that in the [figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39-122&t=OBvM28wKcTRJF0yD-0) and stick to the design as much as possible (e.g., font, colors, images, text, margins). I suggest:

1. Please make your icons align properly. As you are already using flexbox add `align-item: baseline;` would solve this problem

Figma design:

<img width="208" alt="Screenshot 2023-04-01 at 7 30 45 PM" src="https://user-images.githubusercontent.com/44798044/229295971-a25702fe-3f35-49f7-b8f1-5753e37e6f46.png">

Your design:

<img width="320" alt="Screenshot 2023-04-01 at 7 27 14 PM" src="https://user-images.githubusercontent.com/44798044/229295962-ce4a4c27-9498-4f49-af37-88afc4065cbd.png">


2. I have noticed that in your html you have accidentally added `public/index.html` next to `<img` because of which your `fifth icon ` is not showing anymore. Please make sure to fix it.

Reply:
Thank you.